packages/react-percy-ci/src/index.js

```diff
@@ -25,9 +25,15 @@ export default async function run(percyConfig, percyToken) {
   const assets = await compile(percyConfig);
 
   const environment = new Environment();
+  const runtimeEntry = findEntryPath(assets, EntryNames.runtime);
```

Comment:
The runtime entry contains a manifest webpack uses to make sure imports work correctly across entries now that we're splitting some dependencies out into the vendor bundle. The vendor bundle contains `react` and `react-dom`, so that they're not included in multiple scripts.